### DISCUSSION TITLE
API

### DISCUSSION THREAD

**Discussion by reggyamy** [hellosagar/AssigmentHub#11] [Q&A]

**Comment by hellosagar** [comment]
regarding that I haven't made the models public 
but to get the better understanding of the project please have a look at this blog - https://hellosagar.hashnode.dev/assignmenthub-an-app-to-manage-assignments-for-university-professors-aws-amplify-hackathon

In that you can see the GraphQL schema which you can use to generate the schema

I will add the link to  that blog in the readme also later on
